src/v2/view-builder/internals/BaseOktaVerifyChallengeView.js

```diff
@@ -170,18 +170,13 @@ const Body = BaseFormWithPolling.extend({
 
   doCustomURI() {
     this.ulDom && this.ulDom.remove();
+    const IframeView = createInvisibleIFrame('custom-uri-container', this.customURI);
```

Comment:
Is there a scenario where both a `customURI` and `chromeDTC` will be rendered on the same page? If not, think the iframe id should be consistent

Reply:
No, only one will be rendered on the same page.
Do you mean I can use the same iFrame id?

Reply:
Agree we could use the same iframe id, which simplifies the page object a bit since there is only one selector needed to get the iframe

Reply:
Current design (short-term solution) is we do one device probing at a time (always OV probing first if eligible, then Chrome DTC probing if eligible). But in the future we might do all device probings at the same step, that means we could have both a `customURI` and `chromeDTC`, see https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2748980124/Device+Identification+Step+Modeling
If you have a strong opinion I can change them to use the same iFrame id (probably rename the id to be like `iframe-container`), if not I think we can keep them separated.

Reply:
I don't feel strongly. I was thinking there wasn't a lot of downside to keeping the ids different to allow for flexibility for things like you mentioned. @jaredperreault-okta any objections?